WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/ShippingLineDetails.swift

```diff
@@ -52,6 +52,9 @@ struct ShippingLineDetails: View {
                                                  symbol: nil,
                                                  keyboardType: .default)
                                 .accessibilityIdentifier("add-shipping-name-field")
+                                .onTapGesture {
+                                    focusAmountInput = false
```

Comment:
do you have the testing steps for this change?

Reply:
This code was added so that the `test_creat_new_order` UI test can pass on CI. It's a strange issue though, since the test failure only occurs on CI, not on my mac.

BTW, the issue was mentioned in [my original PR](https://github.com/woocommerce/woocommerce-ios/pull/7371#issue-1320359748).

> Here is a very strange issue only happens in UI test on CI. I had to explicitly make [a text field](https://github.com/woocommerce/woocommerce-ios/pull/7371/files#diff-e1ee95623c8f8278d8814fed88be69d800c8ad94be3cbb963a1be585ad3fa979R35) loose keyboard focus by [changing the binding variable](https://github.com/woocommerce/woocommerce-ios/pull/7371/files#diff-e1ee95623c8f8278d8814fed88be69d800c8ad94be3cbb963a1be585ad3fa979R56), for the other "add-shipping-name-field" text field to gain keyboard focus.

Reply:
BTW, you can find my many attempts of fixing this issue in [this buildkite branch builds](https://buildkite.com/automattic/woocommerce-ios/builds?branch=xcode-14-workarounds).

From the test result bundle (you can download one from the artifacts of [this step](https://buildkite.com/automattic/woocommerce-ios/builds/7715#01831096-c84d-40d0-b9ea-7d6307043512)), it appears even thought "add-shipping-name-field" is tapped, it's still "add-shipping-amount-field" that has the keyboard focus, which caused the test to fail.
<img width="545" alt="Screen Shot 2022-09-16 at 9 17 57 AM" src="https://user-images.githubusercontent.com/1101828/190510954-0139e842-ad06-4af9-94c9-b3e3c0557d54.png">